libqfieldsync/offline_converter.py

```diff
@@ -346,6 +350,18 @@ def _convert(self, project: QgsProject) -> None:
         QgsProject.instance().write(str(export_project_filename))
         project.writeProject.disconnect(on_original_project_write)
 
+    def remove_empty_groups_from_project(self, group) -> None:
```

Comment:
```suggestion
    def remove_empty_groups_from_project(self, group: Qgs...) -> None:
```

Please add typings.

Reply:
```suggestion
    def remove_empty_groups_from_project(self, project: QgsProject) -> None:
```

Reply:
Unfortunately, this approach didn't work for nested groups.

To handle it recursively, it's better to pass the group directly. If you have a better idea of how to, please let me know.

Reply:
Great, I missed the recursion. Note to myself to read more carefully!